src/App.tsx

```diff
@@ -60,9 +56,65 @@ export class App extends React.PureComponent<{}, AppState> {
 		})
 	}
 
+	onRemoveRobotIP = (ip: string) => {
+		var array = [...this.state.labCurrentIPs];
+		let StateArray = [...this.state.IPdeleteState];
+		var index = array.indexOf(ip);
+		array.splice(index, 1);
+		StateArray.splice(index, 1);
+		this.setState({labCurrentIPs: array});
+		this.setState({IPdeleteState: StateArray});
+	}
+
+	renderButtons(ip: string) {
+		let IParray = [...this.state.labCurrentIPs];
+		let StateArray = [...this.state.IPdeleteState];
+		let index = IParray.indexOf(ip);
+		if (this.state.IPdeleteState[index]) {
+			StateArray[index] = false;
+			return(
+				<>
+				<Button variant="secondary" disabled>Are you sure you want to delete {ip}?</Button>
```

Comment:
look at the css of other buttons, we should follow one line of design here

Reply:
would love some more explainations on this.